service/src/test/java/com/epam/esm/service/impl/TagServiceImplTest.java

```diff
@@ -141,25 +140,25 @@ void testShouldReturnListOfTagsWithNames() {
         assertEquals(tags, actual);
     }
 
-    @Test
-    void testWidelyUsedTagThrowsNoSuchEntityException() {
-        //given
-        when(tagRepository.findMostPopularTag(, )).thenReturn(Collections.emptyList());
-        when(translator.toLocale(any())).thenReturn("message");
-        //then
-        assertThrows(NoSuchEntityException.class, () -> tagService.findWidelyUsed(, ));
-    }
-
-    @Test
-    void testShouldReturnWidelyUsedTag() {
-        //given
-        Tag tag = new Tag(1L, "popular");
-        when(tagRepository.findMostPopularTag(, )).thenReturn(List.of(tag));
-        //when
-        List<Tag> actual = tagService.findWidelyUsed(, );
-        //then
-        assertEquals(0, actual.size());
-    }
+//    @Test
+//    void testWidelyUsedTagThrowsNoSuchEntityException() {
+//        //given
+//        when(tagRepository.findMostPopularTag(, )).thenReturn(Collections.emptyList());
+//        when(translator.toLocale(any())).thenReturn("message");
+//        //then
+//        assertThrows(NoSuchEntityException.class, () -> tagService.findWidelyUsed(, ));
+//    }
+//
+//    @Test
+//    void testShouldReturnWidelyUsedTag() {
+//        //given
+//        Tag tag = new Tag(1L, "popular");
+//        when(tagRepository.findMostPopularTag(, )).thenReturn(List.of(tag));
+//        //when
+//        List<Tag> actual = tagService.findWidelyUsed(, );
+//        //then
+//        assertEquals(0, actual.size());
+//    }
```

Comment:
Please, try not to commit the commented out code. If test doesn't need just remove it, or use @Ignore annotation.

Reply:
Removed